### ISSUE TITLE
Missing spaces around "=" in keyword argument not reported

### ISSUE THREAD

**Issue by myint** [PyCQA/pycodestyle#323]
@thisch, reported this in hhatto/autopep8#158.

`example.py`:

``` python
def demo_simple_grid(fig):
    grid = AxesGrid(fig, 141,  # similar to subplot(141)
                    nrows_ncols=(2, 2),
                    axes_pad = 0.05,
                    label_mode = "1",
                    )
```

```
$ pep8 --version
1.5.7
$ pep8 example.py
$
```
